api-reference/beta/toc.yml

```
@@ -10901,6 +10901,36 @@ items:
               href: api/unifiedRoleEligibilityScheduleInstance-filterbycurrentuser.md
       - name: Identity and sign-in
         items:
+        - name: Application authentication method policy (preview)
+          items:
+          - name: Overview
+            href: resources/applicationAuthMethodPolicy.md
+          - name: Default app management policy
```

To align with the name change
```suggestion
          - name: Tenant app management policy
```